docs/packages/cli/rpm.rst

```diff
@@ -35,7 +35,7 @@
 * ``--rpm-init FILEPATH``
     - Add FILEPATH as an init script
 * ``--[no-]rpm-macro-expansion``
-    - install-time macro expansion in %pre %post %preun %postun scripts (see: https://rpm.org/user_doc/scriptlet_expansion.html)
+    - install-time macro expansion in %pre %post %preun %postun scripts (see: https://rpm-software-management.github.io/rpm/manual/scriptlet_expansion.html)
```

Comment:
This file is generated from `lib/fpm/package/rpm.rb` which has code to declare flags and help text per flag, so these changes need to be made to that file otherwise they'll be lost the next time the docs are generated.

Example:

https://github.com/jordansissel/fpm/blob/d81a1ad1ac7b47662bcb54e7446f2affb5b3170c/lib/fpm/package/rpm.rb#L157-L159